src/registry_schemas/schemas/change_of_address.json

```diff
@@ -14,55 +14,49 @@
             },
             "allOf": [
                 {
-            "if": {
-                "properties": {
-                    "legalType": {
-                        "const": "CP"
-                    }
-                }
-            },
-            "then": {
-                "properties": {
-                    "offices": {
-                        "registeredOffice": {
-                            "$ref": "https://bcrs.gov.bc.ca/.well_known/schemas/office"
+                    "if": {
+                        "properties": {
+                            "legalType": {
+                                "const": "CP"
+                            }
+                        }
+                    },
+                    "then": {
+                        "properties": {
+                            "offices": {
+                                "registeredOffice": {
+                                    "$ref": "https://bcrs.gov.bc.ca/.well_known/schemas/office"
+                                },
+                                "required": [
+                                    "registeredOffice"
+                                ]
+                            }
                         },
                         "required": [
-                            "registeredOffice"
+                            "offices"
                         ]
-                    }
-                },
-                "required": [
-                    "offices"
-                ]
-            }},
-            {
-            "if": {
-                "properties": {
-                    "legalType": {
-                        "enum": ["BC", "BEN"]
-                    }
-                }
-            },
-            "then": {
-                "properties": {
-                    "offices": {
-                        "registeredOffice": {
-                            "$ref": "https://bcrs.gov.bc.ca/.well_known/schemas/office"
-                        },
-                        "recordsOffice": {
-                            "$ref": "https://bcrs.gov.bc.ca/.well_known/schemas/office"
+                    },
+                    "else": {
```

Comment:
Won't this accidentally require firms to have both registered and records addresses? I think they only have the 1 set.

https://dev.edit.business.bcregistry.gov.bc.ca/FM1083485/correction/?correction-id=150661
![image](https://github.com/user-attachments/assets/90079ae9-83c7-41fc-b691-ce3027be03a2)

https://dev.edit.business.bcregistry.gov.bc.ca/BC0883553/correction/?correction-id=150662
![image](https://github.com/user-attachments/assets/67151baf-5f08-47ca-982a-c60ff8208d17)

Reply:
Firms don't have changeOfAddress, its changeOfRegistration

Reply:
Ah, OK. I thought this was related to corrections.

Resolving.